aedttest/aedt_test_runner.py

```diff
@@ -947,6 +951,9 @@ def execute_aedt(
     aedt_path = get_aedt_executable_path(version)
     command = [aedt_path]
 
+    if int(version) >= 231:
+        os.environ["ANSYSEM_GEOM_KERN_FORCE_OVERWRITE_ORIG_PROJECT"] = "1"
```

Comment:
@beliaev-maksim @boyang2022 what do you think about adding the env. variable there? I have tested and it worked as expected

Reply:
it's sounds good. Is it the same for Linux?

Reply:
@boyang2022 @beliaev-maksim I have to test it in Linux, but I think it should work as well...

Reply:
initially I was opposed to that. Since this can corrupt some customers project.

but since we always copy projects to temp folder, it should not harm. I do not think there is a way to set "output folder" the same as "input" folder